### ISSUE TITLE
Enhancement: Make a browser extension preview Tag Tool to audit all tags (similar to the GTM feature)

### ISSUE THREAD

**Issue by atom-box** [matomo-org/tag-manager#537]
Google GTM has a browser exttension, shown here: 

 A user has suggested an enhanced tag preview tool for MTM. They want to _"be able to see all the different tags and their values there? As an example, if I need to check that the Google Conversion Tracking code is there on a thank you page with the right conversion ID or if I have to make sure that a purchase event from Facebook is triggered...Can I see this same information in preview mode (images below)?"_

![image](https://user-images.githubusercontent.com/22874146/184544228-99de9bcf-41e6-43f7-b858-9f9bb01bdc50.png)

![image](https://user-images.githubusercontent.com/22874146/184544256-aaefeb98-969a-4537-9b1b-46467e42e2c6.png)

**Comment by AltamashShaikh** [comment]
@atom-box We can may be change this
![Screenshot from 2022-08-12 08-36-34](https://user-images.githubusercontent.com/14902277/184277706-02f94ea3-e29d-4097-aa7f-07d072fe4eb9.png)

to this.
![Screenshot from 2022-08-12 08-39-40](https://user-images.githubusercontent.com/14902277/184277941-964b6c9c-5046-4a6c-aa80-2c993015aa67.png)

I have added for prioritisation so that our product teams can check and prioritise it

**Comment by AltamashShaikh** [comment]
we won't audit the tags here, just update the tag name with ID and Event to make it more readable

**Comment by atom-box** [comment]
_Today a user suggested the following improvement for preview. Let me know if this belongs here or deserves its own  feature request._

>  The preview mode is very disorganized, could it be made more user friendly?

**Comment by AltamashShaikh** [comment]
@atom-box may be create a new issue with more detail on what looks dis organised and how would they like to organise it maybe if given a chance

**Comment by Stan-vw** [comment]
Agreed with Altamash that this ticket could use some more 1) what is missing right now, and 2) how would you like it to look.
It'd work for me if you edit the original post here, or add it in comments. We do know that MTM can use a lot of UX fixes, so it'd be great to get community input on this. Things like:
- what is the most difficult to use?
- what is the least clear at the moment?
- what are obvious quick fixes in your view?
- what would be the gold level standard in your view?
- what are other platforms doing well (perhaps with a screenshot or screen recording) and how could we incorporate that (preferably quickly and easily) in MTM?

Hope that helps, can't wait to make MTM really easy yet powerful!
